docs/subnets/network-runner.md

```diff
@@ -335,7 +339,7 @@ It can be used to create your own fully Ethereum-compatible Subnet running on Av
 
 ### subnet-cli
 
-**At this moment the ANR requires an additional tool, [`subnet-cli`](../subnets/subnet-cli.md), to be able to create the necessary configuration to deploy a Subnet in a local custom test-network. Generally, getting a Subnet up and running requires a series of manual steps. We are working hard to make this experience smoother and allow for transparent Subnet definition and creation with improved tooling. Please stand-by.** Suggestions are highly appreciated!
+**At this moment the ANR requires an additional tool, [`subnet-cli`](../subnets/subnet-cli.md), to be able to create the necessary configuration to deploy a Subnet in a local custom test-network. For a smoother experience, we recommend using [avalanche-cli](#../subnets/create-a-local-subnet.md) though, as it hides all this complexity away!** 
```

Comment:
Probably ANR will always require an additional tool, as generating a genesis file for some vm seems to not belong to its domain. I may supress the `At this moment the`, and avoid suggesting subnet-cli as the only one tool.
Probably use `such as` before subnet-cli. 
Really, vm binary installation on avalanchego directory, vm genesis generation, and reinforcing the relationship between vm binary name and the vm name provided to ANR, is needed to be made by another tool or by the experienced user.

Reply:
OK I applied your suggestions.